tests/framework/Optimizers/SimulatedAnnealing/tests

```diff
@@ -13,6 +13,8 @@
   [./ExponentialEggHolder]
     type = 'RavenFramework'
     input = 'ExponentialEggHolder/test_simulatedAnnealing_eggHold.xml'
+    heavy = true
+    max_time = 1500
```

Comment:
Two tests have been added, but not added in the tests file. Please add corresponding files if you want to test them, otherwise remove them.